### ISSUE TITLE
Save button invisible on small screen

### ISSUE THREAD

**Issue by pdqpat** [Robbendebiene/Gesturefy#655]
Editing a gesture on my Asus Transformer (1280x800 10.1 inches) isn't possible. The SAVE button is not shown on the screen even if I reduce it to 50%.
- Operating system: Windows 10 Home  21H2 b19044
- Firefox version: 14.0.2
- Gesturefy version: 3.2.6

![Gesturefly ](https://user-images.githubusercontent.com/5642804/193128228-7dd092d5-50bc-47c4-a53d-3b11f7cbb220.jpg)

**Comment by Robbendebiene** [comment]
Are you using any operating pixel scaling? Because the popup should be around 800x600 pixels. Also you are using text zoom only. that's why it probably doesn't help you.

**Comment by pdqpat** [comment]
No and no.  Here's a screenshot at 30% zoom.
![Gesturefly 30%](https://user-images.githubusercontent.com/5642804/193152133-4f59ac64-88ef-498a-b448-d4a4306ee51b.jpg)

**Comment by Robbendebiene** [comment]
So I was talking about this setting:

![image](https://user-images.githubusercontent.com/13716661/193643171-7cb10c92-816a-4701-8c05-ab6dc285ff53.png)

It seems like you have enabled this, wherefore only the text is scaled on zooming.

Can you please post the output of the bottom part from here: https://codepen.io/Robbendebiene/pen/vYjryZM

**Comment by pdqpat** [comment]
Yes unchecking _Zoom text only_ allows me to reduce the page down to **67%** to see the SAVE button. On a 17" screen, I have to reduce to **80%**.  The editor popup would be just as functional if the gesture size was reduced to works on more screens, especially those who don't want to make images bigger when reading tiny text.

**Comment by Robbendebiene** [comment]
Gesturefy is optimized to work on Screens larger or equal to 800x600 which is basically the smallest desktop resolution. The reason it is not fully responsive is that it doesn't make sense to use Gesturefy on a phone or tablet.

The above numbers (unlike you) clearly state that some sort of display/pixel scaling is enabled. I guess you were not aware of this?

Anyway adjusting the popup to work with some smaller screen sizes shouldn't be that big of a deal.

**Comment by Robbendebiene** [comment]
In the new [Gesturefy 3.2.7](https://github.com/Robbendebiene/Gesturefy/releases/tag/v3.2.7) release the popup now scales by the width of the screen. This might not be the ultimate solution for your problem since you probably suffer from not enough screen height, but you should now at least be able to scroll to the save button.
[rec.webm](https://user-images.githubusercontent.com/13716661/196227125-25c97395-bbcc-470a-9064-142507c455e5.webm)
